From 0e051271bbf1cd87802628f3167faafe7218606f Mon Sep 17 00:00:00 2001
From: Neil Horman <neil.horman@privafy.com>
Date: Sat, 1 Apr 2023 13:07:05 -0400
Subject: [PATCH 12/13] Fix it so we actually stop when we hit an interrupt
 condition

---
 thermal.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/thermal.c b/thermal.c
index 902d7e9..ffff9bd 100644
--- a/thermal.c
+++ b/thermal.c
@@ -190,7 +190,7 @@ static int handle_groupid(struct nl_msg *msg, void *arg)
 static int handle_error(struct sockaddr_nl *sk_addr __attribute__((unused)),
 			struct nlmsgerr *err, void *arg)
 {
-	int rc = (err->error == -NLE_INTR) ? NL_SKIP : NL_STOP;
+	int rc = (err->error == -NLE_INTR) ? NL_STOP : NL_SKIP;
 
 	if (arg && err->error != -NLE_INTR) {
 		log(TO_ALL, LOG_INFO, "thermal: received a netlink error (%s).\n",
-- 
2.33.1

