diff --git a/misc/irqbalance.service.in b/misc/irqbalance.service.in
index 40ff38b..142a9e5 100644
--- a/misc/irqbalance.service.in
+++ b/misc/irqbalance.service.in
@@ -6,8 +6,7 @@ ConditionVirtualization=!container
 ConditionCPUs=>1
 
 [Service]
-EnvironmentFile=@pkgconfdir@/irqbalance.env
-EnvironmentFile=-@usrconfdir@/irqbalance
+EnvironmentFile=-/etc/sysconfig/irqbalance
 ExecStart=/usr/sbin/irqbalance $IRQBALANCE_ARGS
 CapabilityBoundingSet=CAP_SETPCAP
 NoNewPrivileges=yes
